src/Components/ConfirmationScreenComponent.tsx

```diff
@@ -0,0 +1,31 @@
+import { Text, Flex, Button } from "@artsy/palette"
+import { RouterLink } from "System/Router/RouterLink"
+
+export const ConfirmationScreenComponent: React.FC<{
+  title: string
+  subtitle: string
+  buttonTitle: string
```

Comment:
```suggestion
  buttonText: string
```
what do you think about renaming this to `buttonText` 👀?